### PR TITLE
Dig deeper into duplicates

### PR DESCRIPTION
**Pull request by szepeviktor**
This is 🧪 an experiment.

### CODE REVIEW

**Comment by szepeviktor** [comment]
@PrinsFrank There is only 1 `XUA` in the XML. Please help me!
https://www.six-group.com/dam/download/financial-information/data-center/iso-currrency/lists/list-one.xml

**Comment by szepeviktor** [comment]
"We already have EVERYTHING" https://github.com/PrinsFrank/standards/actions/runs/7757939344/job/21158542200?pr=205

File: dev/SpecUpdater.php [hunk 1]
```diff
@@ -31,6 +31,8 @@ public static function update(Event $event): void
                 foreach ($enumFile->fqn::cases() as $existingCase) {
                     if ($enumFile->hasCaseWithValue($existingCase->value) === false) {
                         $enumFile->addCase(new EnumCase($existingCase->name, $existingCase->value, [], true));
+                    } else {
```
Comment (PrinsFrank) [inline dev/SpecUpdater.php:34]: This part of the code adds any cases that were not found in the source code back to the enum. For example:

Enum A at one point contained case B and C.

Now we update enum A from the source, and case B has disappeared. We can't simply drop it from the spec if someone has it hardcoded like this: A::B, so instead we readd it here and mark it as deprecated (see the last parameter that is set to 'true'). This way, we can release the changes in a minor release without having to go up a major release every time a enum case disappears from the source.

Does that make sense?

Comment (PrinsFrank) [reply dev/SpecUpdater.php:34]: To make it even more clear: we are iterating over all cases in an enum, and check if we have found it in the enum cases we pulled from the source. In this case, 99% of all the assertions for this if statement will end up in the else statements as only items that have been removed between spec updates will go into the if branch.

Comment (szepeviktor) [reply dev/SpecUpdater.php:34]: > Does that make sense?

Yes.

I hope `hasCaseWithValue` will never drop **new** cases!

**Pull request closed without merging**